src/Components/CommentComponent/Comment.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react"
+import createComment from "../../helpers/createComment"
+import getAllComments from "../../helpers/getAllComments"
+
+export default class Comment extends Component {
+  state = {
+    comment_arr: [],
+    comment: "",
+    loading: true,
+  }
+
+  commentInputHandler = (e) => {
+    this.setState({ comment: e.target.value })
+  }
+
+  createCommentHandler = async (e) => {
+    const comment = {
+      comment: this.state.comment,
+      createdAt: new Date().getTime(),
+      postID: this.props.postID,
```

Comment:
Why do you need postID in comment structure? It's not being used anywhere.

Reply:
Done